sql.wit.md

```diff
@@ -0,0 +1,66 @@
+# `wasi-sql` API
+
+## Interfaces
+
+```wit
+interface "wasi:sql" {
+    // iterator item type
+    resource row {
```

Comment:
Should be `record`, not `resource`, presumably.

Reply:
You're right~ good catch!